src/update_engine/libcurl_http_fetcher.h

```diff
@@ -101,6 +101,13 @@ class LibcurlHttpFetcher : public HttpFetcher {
     check_certificate_ = check_certificate;
   }
 
+  // Sets HTTP Auth user and password for HTTPS connections.
+  // Copies the given credentials.
+  void set_auth_credentials(const char* auth_user, const char* auth_password) {
+    auth_user_ = auth_user;
```

Comment:
just had to double check that C++ allows directly assigning a `cont char*` to a `std::string`. All seems good.

Reply:
Yes, things happen behind the scenes ;) Hence the comment `Copies`.